schemas/blocks/list_collector_content.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$id": "/blocks/list_collector_content.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "summary": {
+    "type": "object",
+    "properties": {
+      "item_title": {
+        "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
+      },
+      "title": {
+        "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
+      }
+    },
+    "additionalProperties": false,
+    "required": ["item_title", "title"]
+  },
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/identifier"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["ListCollectorContent"]
+      },
+      "content": {
```

Comment:
The ticket suggests this should support both contents and content variants

Reply:
Ok, I've added conditional required `content` or `content_variant` to the rules now. Thanks.